### PR TITLE
[10140] Remove requirement to add Twisted dev team for a PR for review .

### PR DESCRIPTION
**Pull request by adiroiban**
## Scope and purpose

This updates the Github PR template as non team mermber can't update the PR metadata.

In the future we will need a helper that will allow modifying the PR via keywords left in the PR comments.

## Contributor Checklist:

* [x] The associated ticket in Trac is here: https://twistedmatrix.com/trac/ticket/10140
* [x] I ran `tox -e lint` to format my patch to meet the [Twisted Coding Standard](https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html)
* [x] I have created a newsfragment in src/twisted/newsfragments/ (see: [News files](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles))
* [x] The title of the PR starts with the associated Trac ticket number (without the `#` character).
* [x] I have updated the automated tests and checked that all checks for the PR are green.
* [x] I have submitted the associated Trac ticket for review by adding the word `review` to the keywords field in Trac, and putting a link to this PR in the comment; it shows up in https://twisted.reviews/ now.
* [x] The merge commit will use the below format
  The first line is automatically generated by GitHub based on PR ID and branch name.
  The other lines generated by GitHub should be replaced.

```
Merge pull request #123 from twisted/4356-branch-name-with-trac-id

Author: adiroiban
Reviewer: 
Fixes: ticket:10140

Update the GitHub PR template to not ask for Twisted dev teams
as reviewers since non team members can't modify the PR metadata.
```

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -33,7 +32,7 @@ Add mentions of things that are not covered here and are planed to be done in se
 Merge pull request #123 from twisted/4356-branch-name-with-trac-id
 
 Author: <github_username>, <github_usernames_if_more_authors>
-Reviewer: <github_username>, <github_usernames_if_more_reviewers>
+Reviewer: <github_username>, @<github_username_if_more_reviewers>
```
Comment (graingert) [inline .github/PULL_REQUEST_TEMPLATE]: `@s` in commit messages can be *very annoying*

People can fork projects and rebase a lot of commits - firing off thousands of notifications

Comment (adiroiban) [reply .github/PULL_REQUEST_TEMPLATE]: Thanks for the feedback. I was not aware of this. 
Autocompletion is nice... but I can remove the `@` at the end :)

Comment (adiroiban) [inline .github/PULL_REQUEST_TEMPLATE]:
```suggestion
Reviewer: <github_username>, <github_username_if_more_reviewers>
```

**Comment by adiroiban** [comment]
this is ready for a new review. thanks!

**Review by wsanchez — APPROVED** [review]

**Pull request merged**